### ISSUE TITLE
Add v-clip as a directive

### ISSUE THREAD

**Issue by tq-bit** [tq-bit/vitepress-components#6]
```javascript
const vClip = {
  mounted: (el) => {
    el.addEventListener('click', () => {
      const cb = navigator.clipboard;
      const input = el.tagName === 'input' ? el : el.querySelector('input');
      cb.writeText(input.value).then(() => alert('Text copied'));
    });
  },
};
```

**Comment by tq-bit** [comment]
Full App code: 

```javascript
import { createApp } from 'vue';
import App from './App.vue';
import router from './router';

const vClip = {
  mounted: (el) => {
    el.addEventListener('click', () => {
      const cb = navigator.clipboard;
      const input = el.tagName === 'input' ? el : el.querySelector('input');
      cb.writeText(input.value).then(() => alert('Text copied'));
    });
  },
};

const app = createApp(App).use(router);

app.directive('clip', vClip);
app.mount('#app');
```

**Comment by tq-bit** [comment]
solved by `#abfbab3b5eea566922e712e787fafbdb034c2cbf`
